dev/kindly_demo.clj

```diff
@@ -0,0 +1,105 @@
+;; # Wolframite Kindly demo
+
+;; This notebook demonstrates basic usage of [Wolframite](https://github.com/scicloj/wolframite/) in a way that would work in visual tools supporting [Kindly](https://scicloj.github.io/kindly-noted/kindly).
+
+;; It is mostly copied and adpated from [the official Wolframite demo](https://github.com/scicloj/wolframite/blob/main/dev/demo.clj).
+
+(ns kindly-demo
+  (:refer-clojure
+   ;; Exclude symbols also used by Wolfram:
+   :exclude [Byte Character Integer Number Short String Thread])
+  (:require
+   [wolframite.core :as wl :refer [wl]]
```

Comment:
pls use wl/eval instead of wl/wl (fixed in #26)

Reply:
Thanks

Reply:
done